syntax = "proto3";

message AcousticCommand {
    string number = 1; //e.g "1" - "9"
}